pkg/provider/consulcatalog/label.go

```diff
@@ -6,18 +6,24 @@ import (
 
 // configuration Contains information from the labels that are globals (not related to the dynamic configuration) or specific to the provider.
 type configuration struct {
-	Enable bool
+	Enable  bool
+	Connect bool
 }
 
 func (p *Provider) getConfiguration(item itemData) (configuration, error) {
 	conf := configuration{
-		Enable: p.ExposedByDefault,
+		Enable:  p.ExposedByDefault,
+		Connect: p.ConnectByDefault,
```

Comment:
```suggestion
```

Reply:
Is this still a valid suggestion?

Reply:
I do not think so. This was from the time when Tom still thought we could have a "simpler" discovery

Reply:
ACK